docs/application-development-guide.md

```diff
@@ -1146,7 +1146,7 @@ Content is not available
 
 Calls to spawn [Activity Executions](/activities/#activity-execution) are written within a [Workflow Definition](/workflows/#workflow-definitions).
 The call to spawn an Activity Execution generates the [ScheduleActivityTask](/references/commands/#scheduleactivitytask) Command.
-This results in the set of three [Activity Task](/tasks/#activity-task) related Events ([ActivityTaskScheduled](/references/events/#activitytaskscheduled), [ActivityTaskStarted](/references/events/#activitytaskstarted), and ActivityTask[Closed])in your Workflow Execution Event History.
+This results in the set of three [Activity Task](/tasks/#activity-task) related Events ([ActivityTaskScheduled](/references/events/#activitytaskscheduled), [ActivityTaskStarted](/references/events/#activitytaskstarted), and ActivityTask[Closed](/references/events/#activitytaskclosed)) in your Workflow Execution Event History.
```

Comment:
```suggestion
This results in the set of three [Activity Task](/tasks/#activity-task) related Events ([ActivityTaskScheduled](/references/events/#activitytaskscheduled), [ActivityTaskStarted](/references/events/#activitytaskstarted), and one of the Closed status Events) in your Workflow Execution Event History.
```

There is no ActivityTaskClosed Event - this wasn't meant to be a link
this was meant to be a variable placeholder for one of the Closed Event statuses.

However this change needs to be made within the source file, which I think is the `app-dev-context/spawning-activities.md` file -